src/types.ts

```diff
@@ -22,6 +22,12 @@ export interface MonitorOptionsBase {
 
 export interface MonitorOptions<T> extends MonitorOptionsBase {
   parseResult?: (r: Awaited<T>) => any;
+  labeling?: ContextLabeling[];
+}
+
+export interface ContextLabeling {
+  name: string;
+  path: string;
```

Comment:
Why not just value? Why take it from context?

Reply:
oh yeah, it can just be an object for labels

Reply:
Fixed, just passing name and value right now, simpler